### PR TITLE
[WIFI-7221] Add: support for owsub service

### PR DESCRIPTION
**Pull request by dunaev-opsfleet**
Signed-off-by: Dmitry Dunaev <dmitry@opsfleet.com>

### CODE REVIEW

File: docker-compose/traefik/openwifi_selfsigned.yaml [hunk 1]
```diff
@@ -64,6 +71,14 @@ tcp:
       loadBalancer:
         servers:
           - address: "owprov.wlan.local:16005"
+    owprov-restapi:
```
Comment (oblom0v) [inline docker-compose/traefik/openwifi_selfsigned.yaml:74]: This entry is duplicate.

File: chart/values.yaml [hunk 2]
```diff
@@ -38,6 +38,14 @@ owgwui:
 owprovui:
   fullnameOverride: owprovui
 
+# OpenWIFI Subscription (https://github.com/Telecominfraproject/wlan-cloud-userportal/)
+owprov:
```
Comment (oblom0v) [inline chart/values.yaml:42]: I guess this is a typo.

Comment (dunaev-opsfleet) [reply chart/values.yaml]: Correct, fixed it. Thanks for noticing!

File: docker-compose/traefik/openwifi_selfsigned.yaml (hunk 1 above)
Comment (dunaev-opsfleet) [reply docker-compose/traefik/openwifi_selfsigned.yaml]: Yep, copied too much on owsub addition. Thanks for noticing!

**Pull request merged**